docker-compose.yml

```diff
@@ -0,0 +1,108 @@
+version: '2'
+
+services:
+
+  ### Mount Applications ######################################
+
+  applications:
+    image: tianon/true
+    volumes:
+    - ../KojoFitness/:/var/www/html/kojo_fitness.neighborhoods.com
+
+  mysql_data:
+    image: tianon/true
+    volumes:
+    - ./data/mysql:/docker-entrypoint-initdb.d
+
+  pgsql_data:
+    image: tianon/true
+    volumes:
+    - ./data/pgsql:/docker-entrypoint-initdb.d
+
+    ### Kōjō Fitness Container (php-fpm) #########################
+
+  kojo_fitness:
+    build:
+      context: .
+      args:
+      - COMPOSER_TOKEN=z
+      - INSTALL_XDEBUG=true
+      - COMPOSER_INSTALL=false
+    volumes:
+    # Shares AWS credentials to the docker container
+    - $HOME/.aws:/root/.aws
+    - $HOME/.aws:/var/www/.aws
+    volumes_from:
+    - applications
+    environment:
+      SITE_ENVIRONMENT: Local
+      DATABASE_ADAPTER: "pgsql"
+      DATABASE_HOST: pgsql
```

Comment:
Since we're including mysql container stuff, can we have commented out parameters for mysql?

Reply:
Done